conda/config_conda_env_py3.sh

```diff
@@ -18,8 +18,8 @@ else
   exit 1
 fi
 
-echo "=== Installing additional packages for python3 env..."
-source activate ${CONDA_ENV_PY3}
-  pip install caper croo
+#echo "=== Installing additional packages for python3 env..."
```

Comment:
Should we comment these lines or should they just be removed?

Reply:
Let's remove